osu.Game/Scoring/ScoreRank.cs

```diff
@@ -11,6 +11,11 @@ namespace osu.Game.Scoring
 {
     public enum ScoreRank
     {
+        // TODO: reconsider changing later on
+        [LocalisableDescription(typeof(BeatmapsStrings), nameof(BeatmapsStrings.RankD))]
+        [Description(@"F")]
+        F = -1,
```

Comment:
Why go to the trouble of adding this rank *if you're not even using it for the feature it was intended for*? I just checked this branch, the score imports with rank D - so what was the point?

Reply:
Maybe I commit it to my test branch, I remembered that I use `Score,ScoreInfo.Rank = F` at saveReplay()
I will fix it